### PR TITLE
Forum Page

### PR DESCRIPTION
**Pull request by sarinali**
### Description
Created the forum page

### Reason for Change
CFD 96

Also I changed the TabNav component slightly so it supports centring components other than text. Ran into this issue with the SVG so I had to add a view wrapper.

Irrelevant but it took me 45 minutes to position that stupid red circle #notslay :(

### Type of change

- [x] New feature (non-breaking change which adds functionality)

https://github.com/uoftblueprint/centre-for-dreams/assets/74844953/43456adc-c4aa-47a1-8eb1-0b4e87484e42

### CODE REVIEW

File: apps/expo/src/app/forum.tsx [hunk 1]
```diff
@@ -1,13 +1,43 @@
...
+        <View className="ml-4 mr-4 mt-4 flex-1 items-center justify-center">
+          <TabNav>
+            <TabNav.Tab onPress={() => console.log("Tab 1 pressed")}>
+              Feed
+            </TabNav.Tab>
+            <TabNav.Tab onPress={() => console.log("Tab 2 pressed")}>
+              My Posts
+            </TabNav.Tab>
+            <TabNav.Tab onPress={() => console.log("Tab 3 pressed")}>
+              ?
```
Comment (BakerWJ) [inline apps/expo/src/app/forum.tsx:29]: U can now add the icon here using the svg

**Review by BakerWJ — COMMENTED** [review]
looks good -- just add icon and I'll approve

File: apps/expo/src/app/forum.tsx [hunk 2]
```diff
@@ -1,13 +1,59 @@
...
+</svg>
+`;
+
+const redCircleXml = `<svg width="20" height="20" viewBox="-8 0 28 24" height="15" xmlns="http://www.w3.org/2000/svg">
+<circle cx="10" cy="10" r="10" fill="red" />
+</svg>`;
```
Comment (BakerWJ) [inline apps/expo/src/app/forum.tsx:19]: Can you upload svg assets instead?

Comment (sarinali) [reply apps/expo/src/app/forum.tsx:19]: O 😭 K 😭

File: apps/expo/src/app/forum.tsx [hunk 3]
```diff
@@ -1,13 +1,51 @@
...
+          <TabNav>
+            <TabNav.Tab onPress={() => console.log("Tab 1 pressed")}>
+              Feed
+            </TabNav.Tab>
+            <TabNav.Tab onPress={() => console.log("Tab 2 pressed")}>
+              My Posts
+            </TabNav.Tab>
+            <TabNav.Tab onPress={() => console.log("Tab 3 pressed")}>
+              <View className="relative">
+                {/* <SvgXml xml={Bell} width={20} height={20} /> */}
```
Comment (BakerWJ) [inline apps/expo/src/app/forum.tsx:32]: remove

**Review by BakerWJ — APPROVED** [review]

**Pull request merged**